lib/graphql/schema/interface.rb

```diff
@@ -15,10 +15,21 @@ def apply_implemented(object_class)
           end
         end
 
+        def orphan_types(*types)
+          if types.any?
+            @orphan_types = types
+          else
+            all_orphan_types = @orphan_types || []
+            all_orphan_types += super if defined?(super)
```

Comment:
Woah, I have never seen this before! This looks like a good fit for lots of code around here ...